src/main/java/io/micronaut/build/MicronautQualityChecksParticipantPlugin.java

```diff
@@ -84,6 +84,7 @@ private void configureSonar(final Project project) {
                         // Alternatively we could eagerly resolve the checkstyle task,
                         // but we don't want to do this because it makes the build slower
                         sonarQubeExtension.properties(props -> {
+                            props.property("sonar.projectKey", System.getenv("SONAR_TOKEN"));
```

Comment:
This seems like a mistake?

`projectKey` is something like `micronaut-projects_micronaut-core` and `SONAR_TOKEN` is the API key